Spelling error fixes in source code
Index: sg3-utils/src/sg_logs.c
===================================================================
--- sg3-utils.orig/src/sg_logs.c	2010-04-13 10:34:48.827175101 +0530
+++ sg3-utils/src/sg_logs.c	2010-04-13 10:35:26.276203318 +0530
@@ -1410,7 +1410,7 @@
     "aborted by SEND DIAGNOSTIC",
     "aborted other than by SEND DIAGNOSTIC",
     "unknown error, unable to complete",
-    "self test completed with failure in test segment (which one unkown)",
+    "self test completed with failure in test segment (which one unknown)",
     "first segment in self test failed",
     "second segment in self test failed",
     "another segment in self test failed",
@@ -1656,7 +1656,7 @@
                             printf("\n  Threshold temperature = %d C  [IBM "
                                    "extension]", ucp[7]);
                         else
-                            printf("\n  Treshold temperature = <not "
+                            printf("\n  Threshold temperature = <not "
                                    "available>");
                      }
                 }
Index: sg3-utils/src/sg_test_rwbuf.c
===================================================================
--- sg3-utils.orig/src/sg_test_rwbuf.c	2010-04-13 10:35:38.945329591 +0530
+++ sg3-utils/src/sg_test_rwbuf.c	2010-04-13 10:36:01.303176638 +0530
@@ -520,6 +520,6 @@
         if ((0 == ret) && (! do_quick))
                 printf ("Success\n");
         else if (times > 1)
-                printf ("Failed after %d succesful cycles\n", k);
+                printf ("Failed after %d successful cycles\n", k);
         return (ret >= 0) ? ret : SG_LIB_CAT_OTHER;
 }
Index: sg3-utils/src/sg_wr_mode.c
===================================================================
--- sg3-utils.orig/src/sg_wr_mode.c	2010-04-13 10:36:11.005175102 +0530
+++ sg3-utils/src/sg_wr_mode.c	2010-04-13 10:36:22.903176418 +0530
@@ -497,7 +497,7 @@
         if (! force) {
             if ((! (ref_md[off] & 0x80)) && save) {
                 fprintf(stderr, "PS bit in existing mode page indicates that "
-                        "it is not savable\n    but '--save' option given\n");
+                        "it is not saveable\n    but '--save' option given\n");
                 goto err_out;
             }
             read_in[0] &= 0x7f; /* mask out PS bit, reserved in mode select */
